import numpy as np

<<<<<<< HEAD
np.isdtype(1, np.int64)  # type: ignore[arg-type]

np.issubdtype(1, np.int64)  # type: ignore[arg-type]
=======
np.isdtype(1, np.int64)  # E: incompatible type

np.issubdtype(1, np.int64)  # E: incompatible type
>>>>>>> Mental-Health
